dashboard/public/vigenere/vigenere.css

```diff
@@ -0,0 +1,96 @@
+textarea, input, .textarea {
+  font-family: monospace;
+  font-size: 14px;
+}
+
+.btn-group .btn {
+  margin-right: 0;
+}
+
+#left-column fieldset {
+  margin-bottom: 5px;
+  margin-left: 0px;
+}
+
+#left-column > div {
+  /*padding: 5px 10px;*/
+  /*border: solid 1px #5b6770;*/
+  /*box-shadow: 5px 5px 10px #5b6770;*/
+  /*margin-bottom: 10px;*/
```

Comment:
These and the other commented-out styled represent some stylistic choices I'm considering, but haven't yet settled on.